packages/react-sdk/src/user-props/schema.ts

```diff
@@ -1,26 +1,46 @@
 import { z } from "zod";
 import { Asset } from "@webstudio-is/asset-uploader";
 
-const UserPropDb = z.object({
-  id: z.string(),
-  prop: z.string(),
-  value: z.union([z.string(), z.boolean(), z.number()]),
-  assetId: z.optional(z.string()),
-  required: z.optional(z.boolean()),
-});
-export type UserPropDb = z.infer<typeof UserPropDb>;
+// @todo move zod out of sdk
```

Comment:
btw what was the reasoning for moving zod out of SDK?

Reply:
So far my realization was that zod schema's should stay in the place where we fetch the data and potentially transform/stringify/parse it. Basically keep zod at the edge where we receive the data